cmd/crictl/portforward.go

```diff
@@ -36,6 +37,14 @@ var runtimePortForwardCommand = &cli.Command{
 	Name:      "port-forward",
 	Usage:     "Forward local port to a pod",
 	ArgsUsage: "POD-ID [LOCAL_PORT:]REMOTE_PORT",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:    transportFlag,
+			Aliases: []string{"r"},
```

Comment:
Was `r` chosen here because `t` was already taken?

Reply:
Yes, on `exec` and `attach`.

Reply:
Ah bummer.